webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -166,13 +166,16 @@ const loadJobs = () => {
   });
 };
 
-const deleteJob = (jobName) => {
-  const res = confirm('Are you sure to delete the job?');
+const stopJob = (jobName) => {
```

Comment:
Can we gray the stop if the framework already has executiontype STOP or the framework  already  completed?

Reply:
Agree. In addition, I suggest we mark the stopped job with a ‘stopped’, or more precisely, ‘killed’, status.

Reply:
Can you modify the convertState() function accordingly?

Reply:
updated